### ISSUE TITLE
Converting YAML doc with date types to JSON creates invalid JSON doc

### ISSUE THREAD

**Issue by gregbair** [Microsoft/OpenAPI.NET#332]
Microsoft.OpenApi version 1.1.0 -

I'm pulling in a YAML OpenApi doc with the following property in a schema:

```yaml
orderedOn:
    type: string
    description: The date the order was submitted.
    format: date
    readOnly: true
    example: "2018-01-23"
```

When using the serializer  (`doc.Serialize(ms, openApiVersion, format);`, where `ms` is a `MemoryStream`, `openApiVersion` is `OpenApiSpecVersion.OpenApi3_0`, and `format` is `OpenApiFormat.Json`, the JSON output is:

```json
"orderedOn": {
    "type": "string",
    "description": "The date the order was submitted.",
    "format": "date",
    "readOnly": true,
    "example": 2018-01-23T00:00:00.0000000-05:00
}
```

which is invalid JSON (the date is not properly quoted)

**Comment by PerthCharern** [comment]
This is fixed in #322. We will release a patch version by next week.

**Comment by gregbair** [comment]
Thank you, I'll close this then.
